stock_orderpoint_generator/README.rst

```diff
@@ -119,6 +119,6 @@ OCA, or the Odoo Community Association, is a nonprofit organization whose
 mission is to support the collaborative development of Odoo features and
 promote its widespread use.
 
-This module is part of the `OCA/stock-logistics-warehouse <https://github.com/OCA/stock-logistics-warehouse/tree/13.0/stock_orderpoint_generator>`_ project on GitHub.
+This module is part of the `OCA/sale-promotion <https://github.com/OCA/sale-promotion/tree/15.0/stock_orderpoint_generator>`_ project on GitHub.
```

Comment:
@chienandalu I think you have used wrong repo :sweat_smile:

Reply:
Ooops :sweat: